### ISSUE TITLE
Store license diff screenshot in database instead of uploading to github

### ISSUE THREAD

**Issue by rtgdk** [spdx/spdx-online-tools#218]
Currently we upload the diff of the license text on a separate diff repo (see `DIFF_REPO_WITH_OWNER`  in `settings.py`). Instead of uploading to github, we can directly save the static file in the database. Django provides BinaryField to store raw binary data.

Some links, articles and group conversations.
1. https://docs.djangoproject.com/en/3.1/ref/models/fields/#binaryfield 
2. https://groups.google.com/g/django-developers/c/5wRAbViLf1s 
3. https://wiki.postgresql.org/wiki/BinaryFilesInDB

**Comment by Ugtan** [comment]
@rtgdk I don't think so it would be a good idea to use binaryField, I know the number of screenshots would be less but considering the warning on the Django official docs reference [here](https://docs.djangoproject.com/en/3.1/ref/models/fields/#binaryfield). Also, the StackOverflow answer says that we might face some issues related to caching if we use binaryField [reference](https://stackoverflow.com/questions/46460861/django-storing-images-database/46460934). What are your thoughts about this?

**Comment by rtgdk** [comment]
@Ugtan I think we should be good. Having extra columns in sql is fine. We don't expect to have a lot of these. Keeping a separate repo is quite a hassle to setup.

**Comment by Ugtan** [comment]
@rtgdk Okay perfect.

**Comment by jlovejoy** [comment]
@rtgdk - was this resolved and can close issue?

**Comment by goneall** [comment]
@jlovejoy - not resolved yet - a good idea, but probably lower priority to other issues.
